test/setup_test.go

```diff
@@ -364,15 +383,18 @@ func applyAndWaitForApplications(commitID string) {
 	}
 	// want to do "Eventually( Consistently(checkAllAppsSynced, 30sec, 1sec) )"
```

Comment:
Update this comment too.
I'm not sure if the duration of 15 seconds is long enough to examine the stability.  The original setting of 30 seconds did not have a concrete reason, though.

```suggestion
	// want to do "Eventually( Consistently(checkAllAppsSynced, 15sec, 1sec) )"
```

Reply:
The command `argocd app get` itself takes 1 or 2 sec, so actually this process consumes a long time (e.g., over 10 min).  I think it is enough time to confirm convergence.